src/cmd/windows/durable_task_monitor.go

```diff
@@ -0,0 +1,155 @@
+// +build windows
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+
+	"golang.org/x/sys/windows"
+	"jenkinsci.org/plugins/durabletask/common"
+)
+
+// Launches the script in a new session and waits for its completion.
+func launcher(wg *sync.WaitGroup, exitChan chan bool,
+	executable string, args string, resultPath string, outputPath string,
+	launchLogger *log.Logger, scriptLogger *log.Logger) {
+
+	defer wg.Done()
+	defer common.SignalFinished(exitChan)
+
+	var scriptCmd *exec.Cmd
+	var sysAttr windows.SysProcAttr
+
+	sysAttr.CreationFlags = windows.CREATE_NEW_PROCESS_GROUP
+	if executable == "cmd" {
+		scriptCmd = exec.Command(executable)
+		sysAttr.CmdLine = executable + " " + args
+	} else {
+		// Delimiter is a common WITH a space to handle complex arguments that require commas. For example, an argument that includes a method
+		// call with args separated by a comma. The method args can be separated by commas with NO space so they do not accidentally get split
+		scriptCmd = exec.Command(executable, strings.Split(args, ", ")...)
```

Comment:
made the delimiter comma + space (`, `) because I ran into an issue where I needed to call a method in  powershell that required multiple args. Not sure if there's a better way to parse the arg string for the binary.